spec/controllers/medications_controller_spec.rb

```diff
@@ -35,5 +35,23 @@
         expect(subject).to eq('<div class="small_margin_top"><i class="fa fa-bell small_margin_right"></i>Refill reminder email, Daily reminder email</div>')
       end
     end
+
+    describe 'DELETE #destroy' do
+      let(:user) { FactoryGirl.create(:user1) }
```

Comment:
Since only 1 user definition is used across the whole spec, I think it would be nice if every test used this definition placed directly under `describe MedicationsController`.

It really doesn't matter but it's also worth noting that there is a factory simply called "user" and you don't need to use the FactoryGirl class.
```
let(:user) { create(:user) }
```

Reply:
@HashNotAdam thanks for the tip! Glad to save some characters.